docs/source/tutorials/structure_your_input.rst

```diff
@@ -161,11 +107,187 @@ An example for an input directory with videos could look like this:
+that is invariant to these different crops. When using a operation such as grayscale or colorjitter as augmentation,
+it will produce a representation that is invariant to the color information [1]_.
+
+We can use off the shelf augmentations from libraries like `torchvision transforms <https://pytorch.org/vision/stable/transforms.html>`_
+and `albumentations <https://albumentations.ai/docs/>`_ or the ones offered by Lightly SSL's
+`transforms <https://docs.lightly.ai/self-supervised-learning/lightly.transforms.html>`_ submodule while creating our datasets, viz.
+
+.. code-block:: python
+
+    import albumentations as A
+    import torchvision.transforms as T
+    from albumentations.pytorch import ToTensorV2
+
+    ## Torchvision Transforms
+    torchvision_transforms = T.Compose(
+        [
+            T.RandomHorizontalFlip(),
+            T.ToTensor(),
+        ]
+    )
+
+    ## Albumentation Transforms
+    albumentation_transforms = A.Compose(
+        [
+            A.CenterCrop(height=128, width=128),
+            A.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
+            ToTensorV2(),
+        ]
+    )
+
+    ## Lightly Transforms
+    lightly_transform = SimCLRTransform()
 
 .. note::
 
-    Randomly accessing video frames is slower compared to accessing the extracted frames on disk. However,
-    by working directly on video files, one can save a lot of disk space because the frames do not have to
-    be extracted beforehand.
+   You can also create your own augmentations, for more details please refer to :ref:`lightly-custom-augmentation-5` 
+
+Using Transforms for LightlyDataset
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We can use any of these transforms directly while creating a `LightlyDataset` as follows:
+
+.. code-block:: python
+
+    ## Applying Augmentations to a Unlabelled Images Dataset
+    torchvision_aug_image_dataset = LightlyDataset(input_dir='image_dir/', transform=torchvision_transforms)
+    albumentations_aug_image_dataset = LightlyDataset(input_dir='image_dir/', transform=albumentation_transforms)
+    lightly_aug_image_dataset = LightlyDataset(input_dir='image_dir/', transform=lightly_transforms)
+
+    ## Similarly for other data formats (Labeled Image Datasets and Video Datasets)
+
+Using Transforms for PyTorch Datasets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar to native `LightlyDataset`'s we can also pass the transforms while constructing a `LightlyDataset` from a PyTorch Dataset, viz.
+
+.. code-block:: python
+
+    import torchvision
+
+    base = torchvision.datasets.CIFAR10(root="data/torchvision/", download=True)
+
+    torchvision_aug_dataset = LightlyDataset.from_torch_dataset(
+        base, transform=torchvision_transforms
+    )
+    albumentation_aug_dataset = LightlyDataset.from_torch_dataset(
+        base, transform=albumentation_transforms
+    )
+    lightly_aug_dataset = LightlyDataset.from_torch_dataset(
+        base, transform=lightly_transform
+    )
```

Comment:
I think these two sections are a bit overkill as they are quite verbose but do not add much new information. I would replace them and just mention that we can use the transforms for any dataset as shown previously and give a minimal example like:
```
dataset = LightlyDataset(input_dir="image_dir/", transform=torchvision_transform)
# or
dataset = torchvision.datasets.CIFAR10("datasets/cifar10", download=True, transform=albumentations_transform)
```